src/mongo/delegates/Views.class.php

```diff
@@ -202,8 +217,6 @@ public function getViewForResource($resource,$viewType,$context=null)
      */
     public function getViewForResources(Array $resources,$viewType,$context=null)
```

Comment:
I don't see anything in here (or `getViewForResource`) that regenerates the view if the retrieved document is from the wrong spec revision?